README.md

```diff
@@ -1 +1,9 @@
-# ClassicPress-Migration-Plugin
\ No newline at end of file
+# Switch to ClassicPress from WordPress
+
+Switch to ClassicPress from WordPress and say Goodbye to Gutenberg. (ClassicPress Migration Plugin)
+
+https://www.ClassicPress.net/ is a new LTS fork actor of fomous CMS.
+
+Available on GitHub @ https://github.com/ClassicPress/
+
+Sign this petition: https://www.change.org/p/petition-to-wordpress-no-gutenberg-in-wordpress-core/
```

Comment:
Please remove the petition text and link

Reply:
OK to remove the petition text and link